app/app.py

```diff
@@ -21,3 +29,18 @@ def home_page():
     Create the home page for the website.
     """
     return "<p>This is the home page! Make sure the other routes are in different files!</p>"
+
+
+@app.route("/<string:name>/average", methods=["GET"])
+def get_average_undernourishment_by_name(name):
+    """Return the average undernourishment for the given country"""
+    data = db_manager.get_data_by_name(name)
+
+    undernourishments = [row_data.undernourishment for row_data in data]
+
+    return {
+        'data': {
+            'average': sum(undernourishments) / len(undernourishments) \
+                if undernourishments != [] else 0
```

Comment:
also this case only happens when the name is not in our db right? In this case we should just return a error response instead, like 404 or something.

Reply:
make sure to try catch db_manager calls if it might throw an error. If it doesn't then just disregard this.